osu.Game/Storyboards/CommandLoop.cs

```diff
@@ -18,7 +18,12 @@ public class CommandLoop : CommandTimelineGroup
         public readonly int TotalIterations;
 
         public override double StartTime => LoopStartTime + CommandsStartTime;
-        public override double EndTime => StartTime + CommandsDuration * TotalIterations;
+
+        public override double EndTime =>
+            // In an ideal world, we would multiply the command duration by TotalIterations here.
+            // Unfortunately this would clash with how stable handled end times, and results in some storyboards playing outro
+            // sequences for minutes or hours.
+            StartTime + CommandsDuration;
```

Comment:
As mentioned [in the issue thread](https://github.com/ppy/osu/issues/21008#issuecomment-1418871537), there is a separate bug in stable's accounting of the intro/outro time with respect to loops, and if it was to be reproduced here, then this property would need to return just `CommandsDuration` instead. But because it's a nonsensical behaviour and thus clearly a bug, I am disregarding that unless it is proven that doing so breaks anything serious, and going forward with this right now because at least this value has a *semblance* of sanity.